tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -20,3 +24,18 @@ class TestOnDemandCluster(tests.test_resources.test_clusters.test_cluster.TestCl
             "multinode_cpu_cluster",
         ]
     }
+
+    @pytest.mark.level("minimal")
+    def test_rh_status_errors(self, cluster):
```

Comment:
do we need to use a cluster here? instead of tearing down and upping again? If we are just checking the status for a cluster that doesn't exist